staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -133,14 +133,12 @@ var (
 		[]string{"verb", "group", "version", "resource", "subresource", "scope", "component"},
 	)
 	// droppedRequests is a number of requests dropped with 'Try again later' response"
-	//
-	// TODO(wojtek-t): This metric can be inferred both from requestTerminationsTotal as well as
-	// from requestCounter. We should deprecate and remove it.
 	droppedRequests = compbasemetrics.NewCounterVec(
 		&compbasemetrics.CounterOpts{
-			Name:           "apiserver_dropped_requests_total",
-			Help:           "Number of requests dropped with 'Try again later' response",
-			StabilityLevel: compbasemetrics.ALPHA,
+			Name:              "apiserver_dropped_requests_total",
+			Help:              "Number of requests dropped with 'Try again later' response",
```

Comment:
Not sure if there's a precedent for this, but should the metrics help message now indicate that users should now refer to  the `apiserver_requests_total` metric for dropped requests?

Reply:
This is a great suggestion. I think we should amend instrumentation guidelines to do suggest exactly this.

Reply:
https://github.com/kubernetes/community/pull/6565